src/components/Modal/style.css

```diff
@@ -112,6 +112,28 @@
   }
 }
 
+.closeButton {
+  &:hover {
+    .closeButtonLabel {
+      display: block;
+
+      @media (--small-viewport) {
+        /* Safari Mobile tries to show "hover" text on click, so we make sure
+         * that mobile devices don't have any "hover text" */
+        display: none;
```

Comment:
I know the code was only moved here, but curious doesn't this break accessibility? I think this would not allow to announce the button by screen reader anymore?

Reply:
It only applies to the "Close" text label that appears on hover from what I can tell. The button itself has an ARIA label so it still gets announced by the screen reader.